src/app/bitstream-page/clarin-license-agreement-page/clarin-license-agreement-page.component.ts

```diff
@@ -106,6 +107,21 @@ export class ClarinLicenseAgreementPageComponent implements OnInit {
    */
   helpDesk$: Observable<RemoteData<ConfigurationProperty>>;
 
+  /**
+   * The Seznam dataset license content. It is shown directly in this approval page.
+   */
+  LICENSE_NAME_SEZNAM = 'Seznam Dataset Licence';
+
+  /**
+   * The path to the Seznam dataset license content.
+   */
+  LICENSE_PATH_SEZNAM = 'szn-dataset-license.html';
```

Comment:
are we going to hardcode many licenses in the UI ?

Reply:
use `seznam-cz` or similar otherwise it is ambiguous

Reply:
This is the first and hopefully last one hardcoded license.